wscript

```diff
@@ -483,6 +483,12 @@ def configure(conf):
         defines=['_GNU_SOURCE'],
         mandatory=False)
 
+    # Check for backtrace support
+    conf.check_cc(
```

Comment:
Do not use `check_cc` use `check` instead. See https://github.com/waf-project/waf/blob/master/waflib/Tools/c_config.py#L762